src/services/dashboards/all-dashboards/modules/AllDashboardsSelectFilter.vue

```diff
@@ -20,22 +20,21 @@
         <span class="filter-header">Scope</span>
         <p-select-status v-for="(scope, idx) in scopeFilterList"
                          :key="`scope-${idx}`"
-                         v-model="proxyScopeStatus"
+                         v-model="scopeStatus"
                          class="select-desktop"
                          :value="scope.name"
```

Comment:
I think it would be better to use `change` event rather than `watch`.
```
@change="handleChangeScope"
```

Reply:
Thank you for review. It has been applied!